content/docs/quickstart.md

```diff
@@ -7,501 +7,46 @@ next: /docs/about
-   kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.2.0/experimental-install.yaml 
+   kind create cluster
    ```
-   {{< /callout >}}
-   
-2. Port-forward the Argo CD server on port 9999.
-   
-   ```sh
-   kubectl port-forward svc/argocd-server -n argocd 9999:443
-   ```
-
-3. Open the [Argo CD UI](https://localhost:9999/).
 
-4. Log in with the `admin` username and `kgateway` password.
-   
-   {{< reuse-image src="img/argocd-welcome.png" >}}
+2. Deploy the Kubernetes Gateway API CRDs.
 
-5. Create an Argo CD application to install the {{% reuse "docs/snippets/product-name.md" %}} Helm chart. 
-   
-   ```yaml
-   kubectl apply -f- <<EOF
-   apiVersion: argoproj.io/v1alpha1
-   kind: Application
-   metadata:
-     name: kgateway-oss-helm
-     namespace: argocd
-   spec:
-     destination:
-       namespace: gloo-system
-       server: https://kubernetes.default.svc
-     project: default
-     source:
-       chart: gloo
-       helm:
-         skipCrds: false
-         values: |
-           kubeGateway:
-             # Enable {{< reuse "docs/snippets/k8s-gateway-api-name.md" >}} integration
-             enabled: true
-           gatewayProxies:
-             gatewayProxy:
-               disabled: true
-           gloo:
-             disableLeaderElection: true
-           discovery:
-             # For demo purposes, disable discovery
-             enabled: false
-       repoURL: https://storage.googleapis.com/solo-public-helm
-       targetRevision: {{< reuse "docs/versions/n-patch.md" >}}
-     syncPolicy:
-       automated:
-         # Prune resources during auto-syncing (default is false)
-         prune: true 
-         # Sync the app in part when resources are changed only in the target Kubernetes cluster
-         # but not in the git source (default is false).
-         selfHeal: true 
-       syncOptions:
-       - CreateNamespace=true 
-   EOF
-   ```
-   
-6. Verify that the `gloo` control plane is up and running.
-   
-   ```sh
-   kubectl get pods -n gloo-system 
-   ```
-   
-   Example output: 
-   ```txt
-   NAME                                  READY   STATUS      RESTARTS   AGE
-   gateway-certgen-wfz9z                 0/1     Completed   0          35s
-   gloo-78f4cc8fc6-6hmsq                 1/1     Running     0          21s
-   gloo-resource-migration-sx5z4         0/1     Completed   0          48s
-   gloo-resource-rollout-28gj6           0/1     Completed   0          21s
-   gloo-resource-rollout-check-tjdp7     0/1     Completed   0          2s
-   gloo-resource-rollout-cleanup-nj4t8   0/1     Completed   0          39s
-   ```
-
-7. Verify that the `gloo-gateway` GatewayClass is created. You can optionally take a look at how the gateway class is configured by adding the `-o yaml` option to your command.
-   
    ```sh
-   kubectl get gatewayclass gloo-gateway
+   kubectl apply --kustomize "https://github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref=v1.2.1"
```

Comment:
(a) do we need the experimental branch?
(b) do we not have a variable for the version number?